### DISCUSSION TITLE
Ibis + duckdb + OMF parquets

### DISCUSSION THREAD

**Discussion by teemuja** [ibis-project/ibis#9244] [Q&A] [answered]
Hi! Does anyone know how to set duckdb params on ibis duckdb backend when quering remote parquet files with partitions which has some schema issues like below? When using duckdb query setting 'union_by_name=True' does solve the issue below.
'''
IOException: IO Error: Failed to read file "s3://overturemaps-us-west-2/release/2024-05-16-beta.0/theme=buildings/type=building_part[/part-00000-e9b491c1-2173-46a6-bd4d-923baf6a4d4e-c000.zstd.parquet]": schema mismatch in glob: column "class" was read from the original file "s3://overturemaps-us-west-2/release/2024-05-16-beta.0/theme=buildings/type=building[/part-00000-f2bd6ec0-3fbb-47d8-a6dc-2c0213b396b3-c000.zstd.parquet], but could not be found in file "s3://overturemaps-us-west-2/release/2024-05-16-beta.0/theme=buildings/type=building_part[/part-00000-e9b491c1-2173-46a6-bd4d-923baf6a4d4e-c000.zstd.parquet]". Candidate names: id, geometry, bbox, version, update_time, sources, names, level, height, num_floors, min_height, min_floor, facade_color, facade_material, roof_material, roof_shape, roof_direction, roof_orientation, roof_color, building_id
If you are trying to read files with different schemas, try setting union_by_name=True
'''

**Comment by gforsyth** [comment, marked as answer]
Hey @teemuja -- keyword arguments passed to the `read_*` functions are passed through to DuckDB, so you would do `con.read_parquet("path/to/online/blobs", union_by_name=True)`

**Reply by teemuja** [reply]
Wow that was simple. Thanks. Ibis+duckdb rocks! 🙏
